src/AzureClient/AzureSubmissionContext.cs

```diff
@@ -74,16 +74,16 @@ public static AzureSubmissionContext Parse(string inputCommand)
             var timeout = inputParameters.DecodeParameter<int>(ParameterNameTimeout, defaultValue: DefaultExecutionTimeoutInSeconds);
             var pollingInterval = inputParameters.DecodeParameter<int>(ParameterNamePollingInterval, defaultValue: DefaultExecutionPollingIntervalInSeconds);
 
-            var decodedParameters = inputParameters.ToDictionary(
-                item => item.Key,
-                item => inputParameters.DecodeParameter<string>(item.Key));
+            // var decodedParameters = inputParameters.ToDictionary(
```

Comment:
Not sure if this can be deleted? If not, please add a line to explain why this is commented.

Reply:
It can be deleted, yes; that was left in accidentally. Thanks for catching that!